charts/karpenter/templates/rolebinding.yaml

```diff
@@ -18,6 +18,28 @@ subjects:
     name: {{ template "karpenter.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }}
 ---
+# We need this role so that Karpenter can discover the Kubernetes APIServer Endpoints
```

Comment:
Probably just apply the same wording changes from above in here if you make the change